router/voteCallbacks.go

```diff
@@ -257,13 +283,114 @@ func (r *Router) getEnvelopeList(request routerRequest) {
 	if request.ListSize == 0 {
 		request.ListSize = MaxListSize
 	}
-	nullifiers := r.vocapp.State.EnvelopeList(request.ProcessID, request.From, request.ListSize, true)
 	var response types.MetaResponse
-	strnull := []string{}
-	for _, n := range nullifiers {
-		strnull = append(strnull, fmt.Sprintf("%x", n))
+	var err error
+	if response.Envelopes, err = r.Scrutinizer.GetEnvelopes(request.ProcessID, request.ListSize, request.From); err != nil {
+		r.sendError(request, fmt.Sprintf("cannot get envelope list: (%s)", err))
+		return
+	}
+	request.Send(r.buildReply(request, &response))
+}
+
+func (r *Router) getValidatorList(request routerRequest) {
+	var response types.MetaResponse
+	var err error
+	response.ValidatorList, err = r.vocapp.State.Validators(true)
+	if err != nil {
+		r.sendError(request, fmt.Sprintf("cannot get validator list: %v", err))
+		return
+	}
+	request.Send(r.buildReply(request, &response))
+}
+
+func (r *Router) getBlock(request routerRequest) {
+	var response types.MetaResponse
+	if request.Height > r.vocapp.Height() {
+		r.sendError(request, fmt.Sprintf("block height %d not valid for vochain with height %d", request.Height, r.vocapp.Height()))
+		return
+	}
+	response.Block = types.BlockMetadataFromBlockModel(r.Scrutinizer.App.GetBlockByHeight(int64(request.Height)))
+	if response.Block == nil {
+		r.sendError(request, fmt.Sprintf("cannot get block: no block with height %d", request.Height))
+		return
+	}
+	request.Send(r.buildReply(request, &response))
+}
+
+func (r *Router) getBlockByHash(request routerRequest) {
+	var response types.MetaResponse
+	response.Block = types.BlockMetadataFromBlockModel(r.Scrutinizer.App.GetBlockByHash(request.Payload))
+	if response.Block == nil {
+		r.sendError(request, fmt.Sprintf("cannot get block: no block with hash %x", request.Payload))
+		return
+	}
+	request.Send(r.buildReply(request, &response))
+}
+
+func (r *Router) getBlockList(request routerRequest) {
+	var response types.MetaResponse
+	for i := 0; i < request.ListSize; i++ {
+		if int64(request.From)+int64(i) > r.vocinfo.Height() {
+			break
+		}
+		response.BlockList = append(response.BlockList, types.BlockMetadataFromBlockModel(r.Scrutinizer.App.GetBlockByHeight(int64(request.From)+int64(i))))
```

Comment:
Split the lines :)

Are you using golang-ci with the configuration available on the repository? In theory 111 should be the maximum allowed.